WhatsApp/app/src/main/res/layout/activity_main.xml

```diff
@@ -1,12 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:app="http://schemas.android.com/apk/res-auto"
+<LinearLayout
+    xmlns:app="http://schemas.android.com/apk/res"
     xmlns:tools="http://schemas.android.com/tools"
     xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:orientation="vertical"
     android:layout_height="match_parent"
     tools:context=".ui.MainActivity">
 
+    <include
```

Comment:
Why are we adding a content_main that holds fragment navigation when that is already set up in FragmentContainerView